### ISSUE TITLE
agent in Kubernetes, An error occurred while flushing metric on context key

### ISSUE THREAD

**Issue by sunghospark** [DataDog/datadog-agent#1750]
**Describe what happened:
I'm running agent version 6.1.4 on kubernetes version 1.9.7

Following error message is flooding in agent logs.

```
[ AGENT ] 2018-05-29 17:29:46 UTC | INFO | (context_metrics.go:76 in Flush) | An error occurred while flushing metric on context key '66663fb455bb9bad5263897052b17d4f': Rate value is negative, discarding it (the underlying counter may have been reset)
```

**Describe what you expected:**

Would love to know what the implication of this error is and how to mitigate this.

**Steps to reproduce the issue:**
**Additional environment details (Operating System, Cloud provider, etc):**

Kubernetes cluster version 1.9.7 provisioned with KOPS in AWS.
Datadog agent version 6.1.4

**Comment by olivielpeau** [comment]
Hi @sunghospark!

#1598 added more readable info on this log message. The PR is included in versions `6.2.0` and up of the Agent. Therefore, could you update your Agents to have more readable information on what's causing these messages? Thanks!

**Comment by sunghospark** [comment]
Looks better on 6.2.1,  thanks for quick answer and sorry for reporting on old bug, thought I updated to the latest but I was wrong.
